js_modules/dagit/packages/core/src/launchpad/LaunchpadRoot.tsx

```diff
@@ -28,16 +65,28 @@ export const LaunchpadRoot: React.FC<{repoAddress: RepoAddress}> = (props) => {
     return <Redirect to={`/workspace/${repoPath}/pipeline_or_job/${pipelinePath}`} />;
   }
 
-  return <LaunchpadAllowedRoot pipelinePath={pipelinePath} repoAddress={repoAddress} />;
+  return (
+    <LaunchpadAllowedRoot
+      launchpadType="job"
+      pipelinePath={pipelinePath}
+      repoAddress={repoAddress}
+    />
+  );
 };
 
+// ########################
+// ##### LAUNCHPAD ALLOWED ROOT
```

Comment:
This is out of scope for this PR but I don't quite follow why this is called "Launchpad Allowed Root"

Reply:
Yeah that's a historical name, I just left as is.